app/services/backup.rb

```diff
@@ -0,0 +1,234 @@
+# frozen_string_literal: true
+
+class Backup
+  include Rails.application.routes.url_helpers
+
+  BACKUP_DIR = Rails.root.join('tmp', 'backup')
+  ZIP_OUTPUT_DIR = BACKUP_DIR.join('zip')
+  private_constant :BACKUP_DIR, :ZIP_OUTPUT_DIR
+
+  def initialize(user)
+    @user = user
+  end
+
+  def create
+    generate_json_files
+    copy_contents
+    generate_zip_file
+    remove_json_files # cleanup json directories
```

Comment:
抽象化するためにメソッドに切り出してそれらしい名前をつけたのですから、コメントは不要
```suggestion
    remove_json_files
```